test/ms.context.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const metaschema = require('metaschema');
+const metatests = require('metatests');
+
+const { options, config } = require('../lib/metaschema-config/config');
+
+const { MetaschemaError, SchemaValidationError } = metaschema.errors;
+
```

Comment:
Please add a positive test, to ensure that it is possible to use `context` args in actions.